templates/dashboard/base.html

```diff
@@ -101,7 +101,7 @@
               </p>
               <ul>
                 {% if perms.product.view_product %}
-                  <li class="{% block menu_products_class %}{% endblock %}">
+                  <li class="{% block menu_products_type %}{% endblock %}">
```

Comment:
The word "class" in this block's name refers to the CSS class and seem to follow the rule `menu_[model]_class`. Therefore this should remain unchanged. Please also make sure that templates or styles that use this class work properly.